README.md

```diff
@@ -22,9 +22,20 @@ In the specifications in the above table of contents, the keywords "MUST", "MUST
 
 To provide context for users the following section gives example use cases for each part of the spec.
 
-## Filesystem Bundle & Configuration
+#### Application Bundle Builders
 
-- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runtime.
+Application bundle builders can create a [bundle](bundle.md) directory that includes all of the files required for launching an application as a container.
+The bundle contains OCI [configuration files](config.md) where the builder can specify host-independent details such as [which executable to launch](config.md#process-configuration) and host-specific settings such as [mount](runtime-config.md#mount-configuration) locations, [hook](runtime-config.md#hooks) paths, Linux [namespaces](runtime-config-linux.md#namespaces) and [cgroups](runtime-config-linux.md#control-groups).
+Because the configuration includes host-specific settings application bundle directories copied between two hosts may not work and will require configuration adjustments.
```

Comment:
“settings application” → “settings, application”.  And I'd replace “may not work and will require configuration adjustments” with “may require configuration adjustments” (to soften the “will”), because there's a chance that it does work as copied.